### ISSUE TITLE
Migrations should always use settings.ZOMBODB_ELASTICSEARCH_URL not its value

### ISSUE THREAD

**Issue by calinrada** [vintasoftware/django-zombodb#17]
* django_zombodb version: 0.1.4
* Postgres version: 10.7
* ZomboDB version: 10-1.0.3
* Django version: 2.1.7
* Python version: 3.6.4
* Operating System: Ubuntu 16.04

### Description

When we generate migrations, the result is:
```python
migrations.AddIndex(
    model_name='xmodel',
    index=django_zombodb.indexes.ZomboDBIndex(
        fields=['name', 'slug'], 
        name='xmodel_slug_f88d55_zombodb', 
        url='http://127.0.0.1:9200/'
    ),
),
```

The value of ***url*** kwarg should be ***settings.ZOMBODB_ELASTICSEARCH_URL*** because on different environments we might have different ElasticSearch URLs. Running migrations in production or staging/qa, will fail because ES has a different URL.

This should be the correct output:

```python
migrations.AddIndex(
    model_name='xmodel',
    index=django_zombodb.indexes.ZomboDBIndex(
        fields=['name', 'slug'], 
        name='xmodel_slug_f88d55_zombodb', 
        url=settings.ZOMBODB_ELASTICSEARCH_URL
    ),
),
```

**Comment by fjsj** [comment]
Hi @calinrada , thanks catching this.

While that's a problem real issue, solution is not trivial because it's not recommended to use a value that can change in migrations, like one from settings, otherwise the migrations will not be future-proof. I'll think on a solution and implement it in the next couple of days.

**Comment by fjsj** [comment]
Solved by 6eebe1663ca79a3a721b999c3f130dfc62ca9435
Version 0.2.0 launched at PyPI.

The solution was simply to always read 
`ZOMBODB_ELASTICSEARCH_URL` from settings. The `url` parameter at `
ZomboDBIndex` was removed. This can introduce problems if you change 
`ZOMBODB_ELASTICSEARCH_URL` and already have a migrated `ZomboDBIndex`, but I guess this shouldn't happen frequently. Docs were updated to warn about this case (see warning at [Creating an index](https://django-zombodb.readthedocs.io/en/latest/integrating.html#creating-an-index)).

So if you makemigrations locally and run migrate on production, it should work fine.

@calinrada thanks again for the report. Please let me know if you still have the issue. Closing for now.
